### PR TITLE
Update grafana to 2023-09-01

### PR DESCRIPTION
**Pull request by yjsong11**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/SaasSubscriptionDetails.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        internal SaasSubscriptionDetails(string planId, string offerId, string publisherId, SubscriptionTerm term)
+        {
+            PlanId = planId;
+            OfferId = offerId;
+            PublisherId = publisherId;
+            Term = term;
+        }
+
+        /// <summary> The plan Id of the SaaS subscription. </summary>
+        public string PlanId { get; }
```
Comment (archerzz) [inline sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/SaasSubscriptionDetails.cs:32]: Same question. What about those IDs?

File: sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 
 ### Features Added
+- Configure grafana enterprise
+- Configure SMTP
+- Configure grafana plugins
+- Configure grafana major version
+- Specify SKU (pricing plan)
+- Support for private endpoint and managed private endpoint
 
 ### Breaking Changes
```
Comment (archerzz) [inline sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md:13]: remove empty sections

File: sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md [hunk 3]
```diff
@@ -1,14 +1,21 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.0-beta.2 (2023-11-09)
```
Comment (archerzz) [inline sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md:3]: release date is already passed, need to set a future date

**Review by archerzz — COMMENTED** [review]
Generally, it looks good. Need to confirm the formats of a couple of ID properties.

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/GrafanaAvailablePlugin.cs [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        /// <param name="pluginId"> Grafana plugin id. </param>
+        /// <param name="name"> Grafana plugin display name. </param>
+        internal GrafanaAvailablePlugin(string pluginId, string name)
+        {
+            PluginId = pluginId;
+            Name = name;
+        }
+
+        /// <summary> Grafana plugin id. </summary>
+        public string PluginId { get; }
```
[the remark this answers is not in the document]
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/GrafanaAvailablePlugin.cs:28]: This one is a normal string, not resource Id or GUID

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/SaasSubscriptionDetails.cs (hunk 1 above)
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/SaasSubscriptionDetails.cs:32]: This one is a normal string, not resource Id or GUID

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/EnterpriseConfigurations.cs [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+        /// <param name="marketplacePlanId"> The Plan Id of the Azure Marketplace subscription for the Enterprise plugins. </param>
+        /// <param name="marketplaceAutoRenew"> The AutoRenew setting of the Enterprise subscription. </param>
+        internal EnterpriseConfigurations(string marketplacePlanId, MarketplaceAutoRenew? marketplaceAutoRenew)
+        {
+            MarketplacePlanId = marketplacePlanId;
+            MarketplaceAutoRenew = marketplaceAutoRenew;
+        }
+
+        /// <summary> The Plan Id of the Azure Marketplace subscription for the Enterprise plugins. </summary>
+        public string MarketplacePlanId { get; set; }
```
[the remark this answers is not in the document]
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/Models/EnterpriseConfigurations.cs:28]: This one is a normal string, not resource Id or GUID

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/ManagedPrivateEndpointModelData.cs [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+        }
+
+        /// <summary> Provisioning state of the resource. </summary>
+        public GrafanaProvisioningState? ProvisioningState { get; }
+        /// <summary> The ARM resource ID of the resource for which the managed private endpoint is pointing to. </summary>
+        public string PrivateLinkResourceId { get; set; }
+        /// <summary> The region of the resource to which the managed private endpoint is pointing to. </summary>
+        public string PrivateLinkResourceRegion { get; set; }
+        /// <summary> The group Ids of the managed private endpoint. </summary>
+        public IList<string> GroupIds { get; }
```
[the remark this answers is not in the document]
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/ManagedPrivateEndpointModelData.cs:63]: This one is a normal string, not resource Id or GUID

File: sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md (hunk 3 above)
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md]: Thanks, fixed to 2023-11-15

File: sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md (hunk 2 above)
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/CHANGELOG.md]: Removed, thanks!

File: sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/ManagedPrivateEndpointModelData.cs [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+            RequestMessage = requestMessage;
+            ConnectionState = connectionState;
+            PrivateLinkServiceUri = privateLinkServiceUri;
+            PrivateLinkServicePrivateIP = privateLinkServicePrivateIP;
+        }
+
+        /// <summary> Provisioning state of the resource. </summary>
+        public GrafanaProvisioningState? ProvisioningState { get; }
+        /// <summary> The ARM resource ID of the resource for which the managed private endpoint is pointing to. </summary>
+        public string PrivateLinkResourceId { get; set; }
```
[the remark this answers is not in the document]
Comment (yjsong11) [reply sdk/grafana/Azure.ResourceManager.Grafana/src/Generated/ManagedPrivateEndpointModelData.cs]: Thanks - Changed to ResourceIdentifier

**Review by archerzz — APPROVED** [review]
@yjsong11 You need to update the API signature. See the error logs:
> run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions). 

Others look good to me.

**Comment by yjsong11** [comment]
Fixed the API signature, ready to merge. Thanks!

**Comment by archerzz** [comment]
> Fixed the API signature, ready to merge. Thanks!

@yjsong11 merged~

**Pull request merged**